src/api/mapping.ts

```diff
@@ -15,7 +20,39 @@ mappingRouter.post('/', (req, res) => {
   }
   domainKeys.push(mappingObject)
   setData('mappings', domainKeys)
-  res.json(mappingObject)
+  const projectFolder = `${req.body.domain + req.body.subDomain}`
+  const prodConfig = {
+    apps: [
+      {
+        name: `${projectFolder}`,
+        script: './build/app.js',
+        instances: 1,
+        autorestart: true,
+        watch: false,
+        max_memory_restart: '1G',
+        env_production: {
+          NODE_ENV: 'production',
+          PORT: parseInt(req.body.port, 10)
+        }
+      }
+    ]
+  }
+  const projectPath = '~/projects'
+  exec(`
+    mkdir -p ${projectPath}
+    mkdir ${projectPath}/${projectFolder}
+    git init ${projectPath}/${projectFolder}
+    cp ${path.join(
+    __dirname,
+    '../../scripts'
+  )}/post-receive ${projectPath}/${projectFolder}/.git/hooks/
```

Comment:
this is a little confusing, could you create a variable? `const postReceiveScriptPath = ....` and then use the variable here.